src/components/Nav.vue

```diff
@@ -3,14 +3,14 @@
     <nav>
       <div class="nav-wrapper blue darken-1">
         <div class="logo">
-          <div class="brand-logo center">
+          <router-link to="/" class="brand-logo center">
```

Comment:
Redirect directly to `PopularRepositories`
```suggestion
          <router-link :to="{ name: 'PopularRepositories' }" class="brand-logo center">
```